tme/transformer.go

```diff
@@ -0,0 +1,96 @@
+package tme
+
+import (
+	"encoding/base64"
+	"encoding/xml"
+
+	"github.com/Financial-Times/tme-reader/tmereader"
+	"github.com/pborman/uuid"
+	"github.com/pkg/errors"
+)
+
+type Transformer struct {
+}
+
+func (*Transformer) UnMarshallTaxonomy(contents []byte) ([]interface{}, error) {
+	taxonomy := Taxonomy{}
+	err := xml.Unmarshal(contents, &taxonomy)
+	if err != nil {
+		return nil, err
+	}
+	var interfaces []interface{}
+	if len(taxonomy.TermsC) > 0 {
+		interfaces = make([]interface{}, len(taxonomy.TermsC))
+		for i, d := range taxonomy.TermsC {
+			interfaces[i] = d
+		}
+	} else {
+		interfaces = make([]interface{}, len(taxonomy.TermsT))
+		for i, d := range taxonomy.TermsT {
+			interfaces[i] = d
+		}
+	}
+
+	return interfaces, nil
+}
+
+func (*Transformer) UnMarshallTerm(content []byte) (interface{}, error) {
+	return nil, errors.New("Not Implemented")
+}
+
+func transformConcept(tmeTerm Term, endpoint string) BasicConcept {
```

Comment:
It seems to me that even though these functions are unexported, they also have to have a receiver (*Transformer)